Screenshot saved to: /Users/vamsikolla/Desktop/Projects/Darfts/Python-Selenium-POM/assets/screenshots/Step_01_Login_error_user_2025-04-18_12-20-55.png
